### DISCUSSION TITLE
Imported CSV Spreadsheet to Budibase DB, "Unable to import data" error

### DISCUSSION THREAD

**Discussion by cfbauer** [Budibase/budibase#9322] [Help]
Add Source > Budibase DB > Import CSV gives me the red toast error popup: [object object]

Once I do this, the correct columns are created in the Budibase DB. Then in the top right of the Budibase DB I go to "Import > CSV to Import" and select mine, I get a "Unable to import data" error.

Here's an example of my data, there are 16k rows:

```
Date | Exercise | Category | Weight (kgs) | Reps
-- | -- | -- | -- | --
07/01/17 | Overhead Press | Shoulders | 20.41 | 10
07/01/17 | Overhead Press | Shoulders | 24.95 | 5
07/01/17 | Overhead Press | Shoulders | 34.02 | 3
07/01/17 | Overhead Press | Shoulders | 43.09 | 1

```

I did try removing the date column in case that was causing problems, and reducing the total rows to ~4k to see if that helped. No luck.

**Comment by melohagan** [comment]
Hey @cfbauer 

Could you please share some screenshots, including the full error message which you can find in the dev console (Fn+F12 on Chrome)
